vllm/model_executor/layers/attention/attention.py

```diff
@@ -17,11 +17,27 @@ class Attention(nn.Module):
 
     This class takes query, key, and value tensors as input. The input tensors
     can either contain prompt tokens or generation tokens.
+
+
+    If the input tensors contain prompt tokens, the layout is as follows:	
+    |<---------------------- num_valid_tokens ---------------------->|	
+    |<--------------- num_prompt_tokens -------------->|	
+    |<--prompt_0-->|<--prompt_1-->|...|<--prompt_N-1-->|<--padding-->|	
+
+    Otherwise, the layout is as follows:	
+    |<------------------ num_valid_tokens ------------------->|	
```

Comment:
nit: probably we should also include layout for hybrid batches?

Reply:
that will come in the following PRs!